### PR TITLE
Prepare v0.64.0 release

### PR DESCRIPTION
**Pull request by ashutosh-narkar**

### CODE REVIEW

**Review by johanfylling — DISMISSED** [review]
LGTM 👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,15 +3,18 @@
...
 
-The `go` stanza of OPA's `go.mod` had become out of sync: Projects importing OPA as a Go library would see their `go` stanza moved to 1.21 (if it wasn't that before).
-This was introduced with v0.63.0, but the main branch's `go.mod` still claimed to be a `go 1.20` compatible.
-Now, it states the true state of affairs: OPA, used as Go dependency, requires at least Go 1.21, and thus works with all officially supported Go versions (1.21.x and 1.22.x).
+> **_NOTES:_**
+>
+> * The minimum version of Go required to build the OPA module is **1.21**
 
-#### Breaking Change
+This release contains a breaking change in the behaviour of the `Discovery` plugin, mix of features, a new builtin function (`json.marshal_with_options()`),
```
Comment (anderseknert) [inline CHANGELOG.md:12]: It looks odd to me to start off with "this release contains a breaking change in..." and then have that followed up with features. I'd turn that around, or just remove it as it's prominently highlighted below anyway.

Comment (ashutosh-narkar) [reply CHANGELOG.md]: Removed it.

Comment (anderseknert) [reply CHANGELOG.md]: Thanks!

**Review by anderseknert — APPROVED** [review]

**Pull request merged**